Cloud Application Manager/Automating Deployments/google-deployment-manager.md

```diff
@@ -150,6 +151,88 @@ and will be deleted when the instance is terminated.
 
 ![Resources deployed with Google Deployment Manager](../../images/cloud-application-manager/google-deployment-manager/resources.png)
 
+### Example Template
+
+The following template deploys a single virtual machine instance with a firewall rule.
+
+To use this template in a Template Box, you need to define a mandatory variable named `machineType` with value of a valid
+Google Compute Engine virtual [machine type](https://cloud.google.com/compute/docs/machine-types).
+Here are some possible values to define it as an Option type variable: `f1-micro,g1-small,n1-standard-1`
+
+**Important to note** that in the following template, the `name` property of each resource gets a derived value,
+`{{env['deployment']}}` as a prefix, which contains the deployment-ID, and therefore gets a new distinct value
+each time a new Instance is deployed. If we used static names for the resources, a second deployed Instance
+would update the already existing resource, instead of creating a new one, and would delete the only resource
+when terminated, leaving the remaining Instance referring to a non-existing resource, and therefore fail to
+terminate in CAM. And mind the dire consequences of unintentionally deleting resources.
```

Comment:
```suggestion
terminate in CAM. And also bear in mind the direct consequences of unintentionally deleting resources.
```

Reply:
Now it sounds:
`And also bear in mind the dire threat of unintentionally deleting resources.`